messages/status.json

```diff
@@ -0,0 +1,15 @@
+{
+  "examples": [
```

Comment:
How about we consolidate these examples into just two?  I also changed the alias of the production org to "prodOrg" (it was "devHub" previously, which is mileading):

    "sfdx force:org:status --sandboxname DevSbx1 --setalias MySandbox -u prodOrg",
    "sfdx force:org:status --sandboxname DevSbx1 --wait 45 --setdefaultusername -u prodOrg"